build.gradle.kts

```diff
@@ -19,19 +19,21 @@ plugins {
 apply(plugin = "org.openapi.generator")
 
 val mapstructVersion = "1.5.5.Final"
-val postgresqlVersion = "42.6.0"
+val postgresqlVersion = "42.7.2"
 val kotlinLoggingVersion = "3.0.5"
-val springdocOpenapiVersion = "2.2.0"
+val springdocOpenapiVersion = "2.3.0"
 val awaitilityVersion = "4.2.0"
+val wiremockVersion = "3.4.1"
+val jsonWebTokenVersion = "0.12.5"
+val nimbusJwtVersion = "9.37.3"
 
-ext["logback.version"] = "1.4.14"
-ext["jackson-bom.version"] = "2.16.0"
+ext["jackson-bom.version"] = "2.16.1"
 
 allOpen {
   annotations(
     "javax.persistence.Entity",
     "javax.persistence.MappedSuperclass",
-    "javax.persistence.Embeddable"
+    "javax.persistence.Embeddable",
```

Comment:
Linting rules now seems to apply to the gradle build, hence the need for a comma after the last item in a collection!